string name
uint8 move
float32 speed
uint8 STOP = 0
uint8 START = 1
uint8 STAY = 2
---
uint8 status
uint8 SUCCESS = 0
uint8 INVALID_NAME = 1
uint8 ALREADY_STOPPED = 2
uint8 ALREADY_STARTED = 3
uint8 SPEED_OUT_OF_RANGE = 4
uint8 UNKNOWN_ERROR = 5
